app/views/hyrax/homepage/_how_to_use.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="col-md-2"></div>
+<a href="<%= Hyrax::Engine.routes.url_helpers.help_path %>">
+  <div class="col-xs-4 col-sm-2 homepage-box">
```

Comment:
Is there a reason you went with smaller boxes in the center? I thought this was supposed to be full-width, so it should be 4 `col-sm-3` divs. Then I would think we'd drop to 2 wide on mobile, so `col-xs-6`. I mostly just want to get this consistent, so let's decide on one and I can fix my PR either way.

Reply:
We went with the more centered look.

Reply:
Resolved with Slack convo. Targeting this design https://dp.la/ as it was the base reference.
Can change to full-width later if desired.